### DISCUSSION TITLE
Gamepad input not responding

### DISCUSSION THREAD

**Discussion by ImJoosan** [LizardByte/community#92] [Sunshine] [answered]
Hello, today was my first time using sunshine and it hasn't gone well. I'm using sunshine to cast my pc to my Powkiddy Rgb10 Max 2. It works well without to much screen lag or stuttering. My main issue is that no matter what button I press, it isn't being pressed on my pc. I used https://gamepad-tester.com and sure enough, no input. I'm using RetroOZ 0.60 and Moonlight on the Powkiddy. 

I don't understand why video and audio is being played on my Powkiddy but when I try to use input, theres no response. I'm fairly new to all this so It could just be me being dumb. Any help is appreciated!

**Comment by ReenigneArcher** [comment, marked as answer]
Are you using Windows? https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/usage.html#windows

**Reply by ImJoosan** [reply]
Yeah, why?

**Reply by ReenigneArcher** [reply]
Read the docs, link I shared.

**Reply by ImJoosan** [reply]
Omg, I'm stupid. I need [ViGEmBus](https://github.com/ViGEm/ViGEmBus/releases/latest).
